### PR TITLE
integration test (part II) 

### PR DESCRIPTION
**Pull request by yangyansong-adbe**

### CODE REVIEW

File: code/main/task/taskNode.brs [hunk 1]
```diff
@@ -37,5 +37,7 @@ end function
 
 function _adb_stopTaskNode() as void
     GetGlobalAA()._adb_main_task_node.control = "DONE"
-    GetGlobalAA()._adb_main_task_node = invalid
+    if invalid <> GetGlobalAA()._adb_main_task_nodethen
```
Comment (addb) [inline code/main/task/taskNode.brs:40]: space before then

Comment (addb) [reply code/main/task/taskNode.brs:40]: can we do 
```
if  GetGlobalAA()._adb_main_task_node <> invalid 
```

Comment (yangyansong-adbe) [reply code/main/task/taskNode.brs:40]: It seems like we are unable to do that. For some reason, the above code will throw an exception. Also, putting the constant on the left hand side is the coding standard among multiple programing languages. (see `Condition Format` section of the [C coding standard](https://users.ece.cmu.edu/~eno/coding/CCodingStandard.html)) 

I also think we can follow this standard in Brightscript code. What do you think?

Comment (addb) [reply code/main/task/taskNode.brs:40]: We have it the other way at all the places and I find it standard across our mobile SDKs. If we are unable to fix the exception some other way, then I am fine with this.

Comment (yangyansong-adbe) [reply code/main/task/taskNode.brs]: @addb  Fixed the issue in this commit https://github.com/adobe/aepsdk-roku/pull/54/commits/36da2c47754a6a036ea77bff51783afaf3fde78b

**Pull request merged**